### PR TITLE
CRM-13823 - Status Page work

### PR DESCRIPTION
**Pull request by MegaphoneJon**
OK, I think this PR has all issues fixed, and if so, we can close PR #5973!

### CODE REVIEW

File: CRM/Core/BAO/StatusPreference.php [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+   *
+   * @param array $params
+   *
+   * @return array
+   */
+  public static function create($params) {
+    $statusPreference = new CRM_Core_BAO_StatusPreference();
+
+    // Default severity level to ignore is 0 (DEBUG).
+    if (!$params['ignore_severity']) {
```
Comment (totten) [inline CRM/Core/BAO/StatusPreference.php]: Yeah, test results are looking better. From the current results, it sounds like the current failures probably the situation where 'ignore_severity' is totally unset. The conditional here should probably use `empty($params['ignore_severity'])` or `!isset($params['ignore_severity')`.

**Comment by colemanw** [comment]
@PalanteJon I'm excited to get this merged in! Are you able to do a rebase to get it up-to-date?

**Comment by PalanteJon** [comment]
I rebased last week - does it need another rebase?

**Comment by colemanw** [comment]
Yea unfortunately Github can't automatically merge due to some recent changes in master. Let me know if you're able to rebase and if not I'll give it a shot.

**Pull request merged**